packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx

```diff
@@ -342,6 +344,19 @@ export class GroupedVerticalBarChartBase extends React.Component<
             role="img"
           />,
         );
+      if (pointData.data && !this.props.hideValues && this._barWidth >= 16) {
+        barValuesForGroup.push(
+          <text
```

Comment:
Did you test the text is coming in the middle for large values.

Reply:
The labels can only have a maximum of 6 characters because of the formatting